scubagoggles/rego/Commoncontrols.rego

```diff
@@ -504,22 +543,32 @@ NonCompliantOUs3_1 contains {
     LastEvent.NewValue != "DELETE_APPLICATION_SETTING"
 }
 
+NonCompliantOUs3_1 contains {
+    "Name": OU,
+    "Value": NonComplianceMessage3_1
+}
+if {
+    some OU, settings in input.policies
+    enableChallenge := settings.security_login_challenges.enableEmployeeIdChallenge
```

Comment:
`settings.security_login_challenges.enableEmployeeIdChallenge` isn't actually the setting that corresponds to 3.1. It looks like 3.1 isn't in fact currently supported by the policy API.

Reply:
This did correspond to 3.1:

![image](https://github.com/user-attachments/assets/7821801f-40cd-407b-b649-707dba575665)

but it was removed a month ago for some reason.  The rego code has been updated to reflect the changes to 3.1 made on 11/6.

Reply:
The change was to clarify the actual requirement. Step 4 was the essential bit. Steps 5 and on, "Optionally...," were and still are optional, and have been removed to avoid confusion.